changelog.txt

```diff
@@ -15,6 +15,8 @@ that repo.
 
 ## New Scripts
 
+- `warn-stealers`: warn when creatures that may steal your food, drinks, or items become visible
+- `assign-minecarts`: assign minecarts to hauling routes that don't have one
```

Comment:
```suggestion

```

Reply:
I can't believe I keep fucking this up I'm so sorry